docs/release_notes.rst

```diff
@@ -2,12 +2,80 @@
 PUDL Release Notes
 =======================================================================================
 
-.. _release-v2023.XX.XX:
-
 ---------------------------------------------------------------------------------------
 v2023.XX.XX
 ---------------------------------------------------------------------------------------
 
+Dagster Adoption
+^^^^^^^^^^^^^^^^
+* After comparing comparing python orchestration tools :issue:`1487`, we decided to
+  adopt `Dagster <https://dagster.io/>`__. Dagster will allow us to parallize the ETL,
+  persist datafarmes at any step in the data cleaning process, visualize data
+  depedencies and run subsets of the ETL from upstream caches.
+* We are converting PUDL code to use dagster concepts in two phases. The first phase
+  converts the ETL portion of the code base to use
+  `software defined assets <https://docs.dagster.io/concepts/assets/software-defined-assets>`__
+  :issue:`1570`. We will convert the pandas computations cached in the
+  :mod:`pudl.output.pudltabl.PudlTabl` class to use software defined assets in
+  phase 2 :issue:`1973`.
+* General changes:
+
+  * :mod:`pudl.etl` is now a subpackage that collects all pudl assets into a dagster
+    `Definition <https://docs.dagster.io/concepts/code-locations>`__.
+  * The ``pudl_settings``, ``Datastore`` and ``DatasetSettings`` are now dagster
+    resources. See :mod:`pudl.resources`.
+  * IO to sqlite is now handled using the
+    :mod:`pudl.io_managers.pudl_sqlite_io_manager`.
+    Removes the need for the ``pudl.load`` module.
+  * The ``pudl_etl``  and ``ferc_to_sqlite`` commands no longer support loading
+    specific tables. The commands run all of the tables. Use dagster assets to
+    run subsets of the tables.
+  * The ``--clobber`` argument has been removed from the ``pudl_etl`` command.
+  * New static method :mod:`pudl.metadata.classes.Package.get_etl_group_tables`
+    returns the resources ids for a given etl group.
+  * :mod:`pudl.settings.Ferc2XbrlToSqliteSettings` class now loads all FERC
```

Comment:
Is this supposed to be referring specifictally to the FERC Form 2 settings class? Or should it be `pudl.settings.FercToSqliteSettings`?

Reply:
Good catch! I changed this back to `FercToSqliteSettings`.